huggingface-gpt2/ort_addon/ort_supplement/scripts/run_lm_gpt2.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+num_gpus=4
+use_ort=true
+
+export TRAIN_FILE=/data/wiki.train.tokens
+export TEST_FILE=/data/wiki.test.tokens
+
+RUN_FILE=/workspace/transformers/examples/run_language_modeling_ort.py
+RESULT_DIR=/workspace/results
+
+if [ "$use_ort" = true ] ; then
+    echo "Launching ORT run:"
+    RUN_CMD="mpirun -n ${num_gpus} --allow-run-as-root python $RUN_FILE --ort_trainer --output_dir=$RESULT_DIR/output-ort"
```

Comment:
If ORT's logic is encapsulated in a separate entry script, we probably don't need an option to specify ort_trainer